packages/apputils/src/sanitizer.ts

```diff
@@ -1155,7 +1156,7 @@ export class Sanitizer implements IRenderMime.ISanitizer {
       // Set the "disabled" attribute for <input> tags.
       input: sanitize.simpleTransform('input', { disabled: 'disabled' })
     },
-    allowedSchemes: [...sanitize.defaults.allowedSchemes],
+    allowedSchemes: this._options.allowedSchemes ?? [...sanitize.defaults.allowedSchemes],
```

Comment:
I think that you cannot do that here because this method is used to generate `._options`.